libs/datasets/timeseries.py

```diff
@@ -313,6 +313,13 @@ class MultiRegionDataset:
     # DATE. Rows with identical index values may exist.
     tag: pd.Series = _EMPTY_TAG_SERIES
 
+    @property
+    def regions(self) -> Collection[Region]:
```

Comment:
`timeseries_regions`?
Why not `Set[Region]` ? It looks like docstring is out of sync with method signature before it is merged the first time.